netbox/users/apps.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+
+
+class HomeConfig(AppConfig):
```

Comment:
To keep consistent with the other apps:
```suggestion
class UsersConfig(AppConfig):
```